sdk/core/azure-core-test/src/test_proxy_manager.cpp

```diff
@@ -286,6 +286,7 @@ void TestProxyManager::SetProxySanitizer()
         "st",
         "se",
         "sig",
+        "sv",
```

Comment:
What's motivating this core test change here? Please pull this change out into its own PR, rather than into the storage release PR.

Reply:
Hi, since storage sdks will update sas version in each STG Release, so if the `sv` is not ignored, we have to record the test again in each release, this may cost a lot efforts. so this change is mainly to ignore `sv` query parameter. I've opened a PR [Storage/TestProxy playback mode ignore storage sas version](https://github.com/Azure/azure-sdk-for-cpp/pull/4941) for this.

Reply:
Thanks, please revert the changes to this file from this PR.